src/Microsoft.AspNetCore.Authentication.Facebook/FacebookHandler.cs

```diff
@@ -65,12 +66,12 @@ private string GenerateAppSecretProof(string accessToken)
             }
         }
 
-        protected override string FormatScope()
+        protected override string FormatScope(IEnumerable<string> scopes)
```

Comment:
API break, we can't change public or protected API signatures. Add another overload?

Reply:
But is this actually breaking if the FacebookHandler no longer overwrites the method? The method is still there in the FacebookHandler, and subclasses can still overwrite it. It will just fall back to the implementation from the base class.

Reply:
I don’t think I understand this. The old signature is still there. There’s a `OAuthHandler.FormatScope()` which gets inherited by the FacebookHandler. I added a `OAuthHandler.FormatScope(IEnumerable<string>)` which also gets inherited by the FacebookHandler. So the FacebookHandler has *both* signatures. It just stopped overriding one of those in favor of the other one.

Does that still count as a API signature change? Subclasses can still use and override the previous `FormatScope()` without any changes, or not?

Reply:
I know OAuthHandler.FormatScope() is still there but binary compat requires FacebookHandler.FormatScope() remain as well as it's a distinct public API.

Reply:
Ah okay, I see. I’ll readd the override then to do nothing ^^

Reply:
✔️